howto/anbox/develop-platform.md

```diff
@@ -0,0 +1,84 @@
+        -DCMAKE_INSTALL_LIBDIR=lib/${triplet} \
+        -DCMAKE_INSTALL_PREFIX=/usr examples/minimal/
+    ninja -C build
+
+As a result of the build process you will find a `platform_minimal.so` module in the `build` directory.
+
+[AMS](https://discourse.ubuntu.com/t/about-ams/24321) allows launching Anbox containers in a special development mode which is meant for development tasks, specifically for addon or platform development. The Anbox runtime is instructed, when in development mode, not to terminate the container when it detects failures or other problems.
+
+To try out the minimal platform start a raw container with development mode turned on:
+
+    amc launch --raw --devmode --instance-type=a4.3
+
+If you choose the `nvidia` example you need to also selected an [instance type](https://discourse.ubuntu.com/t/instance-types/17764) which supports GPUs:
+
+    amc launch --raw --devmode --instance-type=g4.3
+
+As result, the command will print out the id of the container. Please take a note of the id as you will need it in the next step
+
+The start of the [raw container](https://discourse.ubuntu.com/t/managing-containers/17763#application-vs-raw) will take a moment as it will run through the full [bootstrap process](https://discourse.ubuntu.com/t/managing-applications/17760#bootstrap) before the container is ready to be used.
+
+Once the container is fully up and running you can copy the `platform_minimal.so` module to it:
+
+    id=<id of the container>
+    triplet=$(dpkg-architecture -qDEB_BUILD_MULTIARCH)
+    amc exec "${id}" mkdir "/usr/lib/${triplet}/anbox/platforms/minimal"
+    lxc file push build/platform_minimal.so "ams-${id}/usr/lib/${triplet}/anbox/platforms/minimal/"
+
+With the platform plugin present inside the container, you can now configure the Anbox runtime to make use of it. For that you have to change the `/var/lib/anbox/session.yaml` configuration file within the container:
+
+    cat << EOF | amc exec "${id}" tee /var/lib/anbox/session.yaml
+    log-level: debug
+    platform: minimal
+    EOF
+
+This will rewrite the Anbox runtime configuration to use the new "minimal" platform instead of the default one.
+
+As last step you have to restart Anbox inside the container
+
+    amc exec "${id}" -- systemctl restart anbox
+
+Once the Anbox runtime got restarted, you can verify from the container system log that it now uses the new "minimal" platform:
+
+    amc logs "${id}"
+
+Inside the log you will fine a line saying
+
+    [ 2022-12-07 14:00:18] [anbox-session] [base_platform.cpp:34@create] Using platform 'minimal'
+
+This shows that the platform was successfully loaded and is in use by Anbox.
+
+When doing development of a new platform you can also directly invoke the Anbox runtime rather than starting it via systemd. For that you first have to stop the `anbox.service` unit
+
+    amc exec "${id}" -- systemctl stop anbox
+
+Now you can start the Anbox runtime directl by running
```

Comment:
```suggestion
When developing a new platform, you can also directly invoke the Anbox runtime rather than start it via `systemd`. For that, stop the `anbox.service` unit first:

    amc exec "${id}" -- systemctl stop anbox

Then start the Anbox runtime directly by running:
```